include/macroni/Translation/MacroniCodeGenVisitorMixin.hpp

```diff
@@ -111,8 +95,9 @@ namespace macroni {
 
             // Check if the macro is an expansion or a parameter, and return the
             // appropriate operation
+            mlir::Operation *op = nullptr;
```

Comment:
Keep returns, wrap the thing in a lambda an invoke it right away. Also drop else.